django-sites/blog/templates/blog/blog_index_page.html

```diff
@@ -19,9 +19,7 @@ <h3 class="post-subtitle">
                   {{ post.specific.intro }}
                 </h3>
               </a>
-              <!-- doesnt work yet - needs fixing:
-              <p class="post-meta">Posted on {{ post.date }}</p>
-              -->
+              <p class="post-meta">Posted on {{ post.specific.date }}</p>
```

Comment:
Well done! Work's on my machine. :)
While we're at it, what how do you feel about making this template language neutral and leaving out the text 'Posted on' and simply displaying the date itself? (esp. since our target system is in German)

Reply:
Will do. :-)

Reply:
Pull request updated.